index.html

```diff
@@ -217,22 +222,28 @@ <h3>{{RTCPeerConnection/addTransceiver()}}</h3>
        <h3>{{RTCRtpSender/setParameters()}}</h3>
          <p>
            [[!WEBRTC]] Section 5.2 describes validation of <var>parameters</var>
-           within {{RTCRtpSender/setParameters()}}. Add the following to the 
+           within {{RTCRtpSender/setParameters()}}. Add the following to the
```

Comment:
I think I prefer s/add/insert/
```suggestion
           within {{RTCRtpSender/setParameters()}}. Insert the following to the
```

Applies throughout.